apps/src/templates/hintsDisplay.jsx

```diff
@@ -0,0 +1,57 @@
+module.exports = React.createClass({
```

Comment:
By convention I think we've been capitalizing our jsx classes (ie. HintsDisplay.jsx), though it might be super difficult to do change this now in git.